test/controllers/task_controller_test.exs

```diff
@@ -12,17 +9,19 @@ defmodule CodeCorps.TaskControllerTest do
   }
 
   @invalid_attrs %{
+    title: nil,
     task_type: "issue",
     status: "nonexistent"
   }
 
-  defp build_payload, do: %{ "data" => %{"type" => "task"}}
-
   describe "index" do
     test "lists all entries", %{conn: conn} do
-      path = conn |> task_path(:index)
-      json = conn |> get(path) |> json_response(200)
-      assert json["data"] == []
+    [task_1, task_2] = insert_pair(:task)
+
+    conn
+    |> request_index
+    |> json_response(200)
+    |> assert_ids_from_response([task_1.id, task_2.id])
```

Comment:
Indentation here is bad. Which editor are you using, and does it have an `.editorconfig` plugin? Should prevent this from happening. 

Feel free to mention it on slack. Someone can guide you through configuring an editor when we get the chance.